README.markdown

```diff
@@ -4523,6 +4523,10 @@ or
  ngx.req.set_uri("/foo", true)
 ```
 
+The optional boolean `binary` argument is used to decide the form of the uri(default to `false`).  If set to true, ngx.req.set_uri will not check safety of the uri.
+For example, ngx.req.set_uri("/foo\r\n/bar", false, false) or ngx.req.set_uri("/foo\r\n/bar", false) will failed while ngx.req.set_uri("/foo\r\n/bar", false, true)  will pass.
```

Comment:
It seem there is 2 space after `while ngx.req.set_uri("/foo\r\n/bar", false, true)`?